test-cases/performance/perf-scylla-cassandra-uniform.yaml

```diff
@@ -0,0 +1,5 @@
+stress_cmd_m: ["cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=100 throttle=12500/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=UNIFORM(1..1000000000)' ",
```

Comment:
when you run both `.yaml` files together, which one of the `stress_cmd_m` will take precedent?

Reply:
if i am not mistaken, latest one